web-react-ts/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "web-react-ts",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@apollo/client": "^3.1.2",
+    "@material-ui/core": "^4.11.0",
+    "@material-ui/icons": "^4.9.1",
+    "@testing-library/jest-dom": "^4.2.4",
+    "@testing-library/react": "^9.3.2",
+    "@testing-library/user-event": "^7.1.2",
+    "@types/jest": "^24.0.0",
+    "@types/node": "^12.0.0",
+    "@types/react": "^16.9.0",
+    "@types/react-dom": "^16.9.0",
+    "@types/react-router-dom": "^5.1.5",
+    "@types/recharts": "^1.8.14",
```

Comment:
looks like you might be missing a ref for

```
    "graphql": "^15.3.0",
    "graphql-tag": "^2.11.0",
```

Reply:
These packages came as dependencies from apollo-client 3.* I use the latest apollo client.

e.g.
[1/4] Why do we have the module "graphql-tag"...?
[2/4] Initialising dependency graph...
[3/4] Finding dependency...
[4/4] Calculating file sizes...
=> Found "graphql-tag@2.11.0"
info Reasons this module exists
   - "@apollo#client" depends on it
   - Hoisted from "@apollo#client#graphql-tag"